### PR TITLE
User/Auth: Increase field size of `login_attempts` (ILIAS 10)

### PR DESCRIPTION
**Pull request by mjansenDatabay**
See: https://mantis.ilias.de/view.php?id=42516

If approved, this schema change should be picked to `trunk` as well.

Depens on: https://github.com/ILIAS-eLearning/ILIAS/pull/8357 (for ILIAS >= 10.x)

### CODE REVIEW

**Review by kergomard — APPROVED** [review]
Thank you very much for the PR @mjansenDatabay ! This looks right now and can be merged as soon as #8357 is merged. I will try to keep an eye on it and will merge once this happend, but if you merge the other one and feel like it, don't hesitate to simply merge this one too.

**Comment by mjansenDatabay** [comment]
The base PR is merged.

**Comment by kergomard** [comment]
Thank you very much @mjansenDatabay ! Merged and picked to trunk.

**Pull request merged**
